src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -127,6 +127,13 @@ public class UnsolvedSymbolVisitor extends ModifierVisitor<Void> {
    * type of fields in the parent class of the current class.
    */
   private final Set<String> syntheticTypes = new HashSet<>();
+
+  /**
+   * A mapping of field names to the @ClassGetSimpleName name of the classes in which they are
+   * declared.
```

Comment:
This comment should be more specific: this is the fields of *the class currently being visited and its inner classes* - that is, the fields defined in the same file being visited and not in super classes.